Dockerfile

```diff
@@ -1,7 +0,0 @@
-FROM atomiq/node:onbuild
```

Comment:
The lib template still has a Dockerfile for being able to run a test from a docker container.